### PR TITLE
Digital pack forms

### PR DESCRIPTION
**Pull request by marchibbins**
Markup and CSS, JS to follow.

image

image

image

### CODE REVIEW

File: public/stylesheets/ie-old.min.css [hunk 1]
```diff
@@ -1 +1 @@
...
...m;height:60px;width:60px}.digital-cta{padding-top:24px;padding-bottom:36px;padding-top:12px;padding-bottom:48px}.digital-cta__price{font-size:21px;line-height:27px;text-align:center;text-align:right}.digital-cta__amount{font-size:42px}.digital-cta__payment{color:#767676;text-align:center;text-align:right}.digital-cta__button{margin-top:14px;text-align:center;text-align:left}.basket{padding-top:2px;border-top:1px solid #005689}.basket__detail{border-top:1px dotted #dfdfdf;display:block;margin-top:6px;padding-top:8px}.basket__header{font-weight:900}.basket__smallprint{font-size:13px;line-height:18px;font-family:"Guardian Text Sans Web","Helvetica Neue",Helvetica,Arial,"Lucida Grande",sans-serif;margin-bottom:12px}.review-details--bg{background:#f6f6f6;border-top:1px solid #dfdfdf;padding:10px}.review-details__heading{font-size:16px;line-height:24px;font-family:"Guardian Text Egyptian Web",Georgia,serif;font-weight:bold;float:left;width:140px;text-align:right}.review-details__list{margin-bottom:12px;margin-left:160px}.review-details__detail{display:inline-block;width:140px}.row--items-1 .row__item{width:100%}.global-header,.global-footer{min-width:1020px}.gs-container{min-width:980px}
```
Comment (tudorraul) [inline public/stylesheets/ie-old.min.css]: is this a generated file? if so, can it be generated at build time and not be present in VCS?

File: public/stylesheets/ie9.min.css [hunk 2]
```diff
@@ -1 +1 @@
...
...ze:42px}.digital-cta__payment{color:#767676;text-align:center}@media (min-width: 46.25em){.digital-cta__payment{text-align:right}}.digital-cta__button{margin-top:14px;text-align:center}@media (min-width: 46.25em){.digital-cta__button{text-align:left}}.basket{padding-top:2px}@media (min-width: 46.25em){.basket{border-top:1px solid #005689}}.basket__detail{border-top:1px dotted #dfdfdf;display:block;margin-top:6px;padding-top:8px}.basket__header{font-weight:900}.basket__smallprint{font-size:13px;line-height:18px;font-family:"Guardian Text Sans Web","Helvetica Neue",Helvetica,Arial,"Lucida Grande",sans-serif;margin-bottom:12px}.review-details--bg{background:#f6f6f6;border-top:1px solid #dfdfdf;padding:10px}.review-details__heading{font-size:16px;line-height:24px;font-family:"Guardian Text Egyptian Web",Georgia,serif;font-weight:bold}@media (min-width: 46.25em){.review-details__heading{float:left;width:140px;text-align:right}}.review-details__list{margin-bottom:12px}@media (min-width: 46.25em){.review-details__list{margin-left:160px}}.review-details__detail{display:inline-block;width:140px}.row--items-1 .row__item{width:100%}.global-header,.global-footer,.gs-container{min-width:1020px}
```
Comment (tudorraul) [inline public/stylesheets/ie9.min.css]: dito

File: public/stylesheets/main.min.css [hunk 3]
```diff
@@ -1 +1 @@
...
...igital{background-image:url(/assets/images/backgrounds/bg-digital-pack.png);background-position:bottom right;background-repeat:no-repeat}}@media (min-width: 46.25em){.block--weekly{background-image:url(/assets/images/backgrounds/bg-guardian-weekly.png);background-position:bottom right;background-repeat:no-repeat}}@media (min-width: 46.25em){.block--two-col .block__icon{left:5px;top:36px}.block--two-col .block__title{float:left}.block--two-col .block__info{margin-left:50%;margin-top:3px}.block--two-col .block__footer{left:50%}.block--two-col.block--primary .block__info{padding-bottom:36px}.block--two-col.block--primary .block__footer{bottom:12px}}.membership-promo{background-color:#fff;margin-top:-10px;margin-bottom:-10px}@media (min-width: 46.25em){.membership-promo{background-color:#dfdfdf;margin-top:0;margin-bottom:0}}.membership-promo .block{background:#005689 url(/assets/images/backgrounds/bg-membership-small.jpg) no-repeat;-webkit-box-shadow:none;-moz-box-shadow:none;box-shadow:none;color:#fff;height:240px;padding-top:10px;padding-bottom:10px}@media (min-width: 46.25em){.membership-promo .block{background:#005689 url(/assets/images/backgrounds/bg-membership.jpg) no-repeat;heig
```
Comment (tudorraul) [inline public/stylesheets/main.min.css]: dito

**Comment by tudorraul** [comment]
:+1: LGTM, bar the comment

**Comment by marchibbins** [comment]
@tudorraul yep, that's the plan eventually. Since existing GAE app uploads local files we needed it compiled first, these will be replaced with build scripts to do this as part of new deployment.

**Pull request merged**